frontend/src/scenes/groups/GroupsTabs.tsx

```diff
@@ -9,13 +10,25 @@ export function GroupsTabs(): JSX.Element {
     const { setTab } = useActions(groupsListLogic)
     const { currentTab } = useValues(groupsListLogic)
     const { groupTypes } = useValues(groupsModel)
+    const { groupsAccessStatus } = useValues(groupsAccessLogic)
+
+    const showGroupsIntroductionPage = [
```

Comment:
This is confusing, it makes it look like we show this when for instance the user has no access to groups at all but the `GroupsTabs` component is only mounted when groups are fully enabled and further, at least one group exists, so is this actually being used at all?

Reply:
showGroupsOptions is "fixed" in the other PR to account for when we should show upsell as well.

Complicated business logic leads to complicated code. :shrug: